test/binary_search_tree_test.rb

```diff
@@ -75,9 +74,7 @@ def test_it_can_find_depth_of_each_movie_when_inserting_twelve_movies
   end
 
   def test_it_can_verify_presence_of_score_in_the_tree
-    @movies_list.each do |key, value|
-      result = @tree.insert(key, value[0])
-    end
+    load_movies
 
```

Comment:
I moved this method into the test file, exactly as is, and wrapped it in a `load_movies` method. Now you can call this from your tests, as needed. 

(You don't use it everywhere, so it couldn't be a setup method.)